### ISSUE TITLE
Don't fill queue with requests while the worker is booting

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#1340]
Here's an example failure of the integration test:
[live_backing.txt](https://github.com/mozilla/bugbug/files/4268644/live_backing.txt)

I think what's happening is that we enqueue a lot of requests while the workers are starting, and so we spend a lot of time doing them all.

We should queue jobs only once while the workers are not running.

**Comment by marco-c** [comment]
It should in theory already be working with the `is_running` check, but apparently it isn't (or at least I think it isn't, let's see what happens with https://community-tc.services.mozilla.com/tasks/FdozjNfCRRCy_Gtk5qEFqg).

**Comment by ahal** [comment]
I ran into this too, I fixed it by returning true if status is "queued":
https://github.com/mozilla/bugbug/pull/1319/commits/7262584827fa94041adec7cfa24f14d04b711063#diff-596c6d06758c5d9b7ca20ffba2196bb2R197

I'll roll that change out into a separate PR that fixes this issue when I get a chance.

**Comment by marco-c** [comment]
As part of this, we should also move https://github.com/mozilla/bugbug/commit/791c16ebe0f8a607ab194d477ce602d8ecd7d80b#diff-aec5ad9e6cf04fcf90061c24d3443d57R26 and https://github.com/mozilla/bugbug/commit/791c16ebe0f8a607ab194d477ce602d8ecd7d80b#diff-aec5ad9e6cf04fcf90061c24d3443d57R49 back to 1 second.
